src/main/resources/ready-api-rest-api-spec.json

```diff
@@ -327,6 +327,92 @@
       }
     }
   },
+  "/v1/readyapi/executions/{executionID}/logs": {
```

Comment:
I think we can remove this file and generate directly from the definition on SwaggerHub